### ISSUE TITLE
While restoring cant get original transaction id..

### ISSUE THREAD

**Issue by abdulKarim002** [bizz84/SwiftyStoreKit#193]
### Platform

- [x] iOS

### In app purchase type

- [x] Auto-Renewable Subscription

### Environment

- [x] Sandbox

### Version
ℹ 0.8.5

### Related issues
ℹ Please replace this with references to similar issues (either open or already closed).

### Issue summary
ℹ Can't able to get original transaction id while restoring

### What did you expect to happen
ℹ while restoring I want to get original transaction id.

### What happened instead
ℹ but while restoring I am getting transaction id

**Comment by bizz84** [comment]
@abdulKarim002 for restored products, you can access the `product.transaction.transactionIdentifier` as needed:

```swift
public struct Product {
    public let productId: String
    public let transaction: PaymentTransaction
    public let needsFinishTransaction: Bool
}
public protocol PaymentTransaction {
    var transactionState: SKPaymentTransactionState { get }
    var transactionIdentifier: String? { get }
}
```

<img width="1012" alt="restoretransactionid" src="https://cloud.githubusercontent.com/assets/153167/25805079/364c3c2e-33f6-11e7-90f4-2c5068720725.png">

Are you saying that the transaction id you get is different from the transaction id on the original purchase?

**Comment by bizz84** [comment]
I understand now. Restored transactions have an `original: SKPaymentTransaction` value that should be returned instead of the default one.

```swift
open class SKPaymentTransaction : NSObject {    
    // Only valid if state is SKPaymentTransactionStateRestored.
    @available(iOS 3.0, *)
    open var original: SKPaymentTransaction? { get }
}
```

I'll treat this as a bug to be fixed

**Comment by bizz84** [comment]
Fix available on pod version [0.9.0](https://github.com/bizz84/SwiftyStoreKit/releases/tag/0.9.0).

**Comment by abdulKarim002** [comment]
Thank @bizz84
